data/chemnitz.csv

```diff
@@ -0,0 +1,15 @@
+year,category,type,co2,note
+1989,Gesamt,real,2893,
+1992,Gesamt,real,1854,
+1998,Gesamt,real,2003,
+2002,Gesamt,real,1904,
+2005,Gesamt,real,2200,
+2011,Gesamt,real,1879,
+2012,Gesamt,real,1830,
+2013,Gesamt,real,1780,
+2014,Gesamt,real,1830,
+2015,Gesamt,real,1681,
+2016,Gesamt,real,1656,
+2017,Gesamt,real,1632,last_emissions
+2030,Gesamt,geplant,1891
```

Comment:
Wie wurde das genau gerechnet? In https://chemnitz.de/chemnitz/media/unsere-stadt/umwelt/klimaschutzbericht_2016_2017.pdf, Seite 12 haben wir 5,85 t/EW für das Jahr 2030, wir rechnen 0,9t drauf und multiplizieren das mit 247237, da kommt dann 1669k und nicht 1891k raus.

Reply:
good catch!